lib/elsa/group/manager/worker_supervisor.ex

```diff
@@ -92,12 +96,18 @@ defmodule Elsa.Group.Manager.WorkerSupervisor do
       # This is the DynamicSupervisor created in init
       dynamic_worker_supervisor = ElsaRegistry.whereis_name({registry, :worker_dynamic_supervisor})
 
-      # Synchronously stops the DynamicSupervisor and its children
-      DynamicSupervisor.stop(dynamic_worker_supervisor)
-
-      # Make sure the DynamicSupervisor itself is truly cleaned up from the Supervisor's perspective,
-      # so that it will restart reliably
-      _ = Supervisor.terminate_child(module_supervisor, :worker_dynamic_supervisor)
+      if dynamic_worker_supervisor != :undefined do
```

Comment:
If for some reason the dynamic_worker_supervisor isn't alive at this point (ex. because we got that transient "coordinator not available error" and it prevented creation or killed off the supervision tree), we should skip the rest so that we don't crash out the top-level supervisor process.